src/common/adapters/jira.js

```diff
@@ -1,61 +1,54 @@
-import {
-  $all,
-  $find,
-  $has,
-  $text,
-  $attr,
-} from './helpers';
-
-const TYPES = ['bug', 'chore'];
-
-const normalizeType = (type) => {
-  const sanitizedType = type && type.toLowerCase();
-  if (TYPES.indexOf(sanitizedType) > -1) return sanitizedType;
-  return 'feature';
-};
-
-const ticketPageTitle = (issue) => {
-  if ($has('#summary-form', issue)) { // ticket title is currently being edited
-    return $text('#summary-form #summary', issue);
-  }
+// JIRA adapter
+// It gets the selected issue from the current URL and then it fetches the ticket info
```

Comment:
I would add an empty comment line above this one, update text just a wee bit and include a link to the API documentation:

```javascript
// Jira adapter
//
// This adapter extracts the identifier of the selected issue from the page URL
// and uses the Jira API to retrieve the corresponding ticket information.
//
// https://developer.atlassian.com/cloud/jira/software/rest/